pages/uat_actions/generate_test_run.vue

```diff
@@ -6,7 +6,19 @@
 
     <Alert id="alert" ref="alert"></Alert>
 
-    <b-card title="Generate Test Run" sub-title="Number of plates containing number of positives.">
+    <b-card title="Generate Test Run">
+      <b-card-text>
+        Select the number of plates you want to generate test data for with the number of positive samples per plate.
+        You can add additional sets of plates up to a maximum of {{ maxNumberOfPlates }}.
+      </b-card-text>
+      <b-card-text>
+        N.B. for Beckman testing remember to check the
+        <em>Add to DART</em> box. DART data is not needed for Biosero testing.
+      </b-card-text>
+      <b-card-text>
+        The
+        <em>Generate</em> button will start the creation of the data, and you will be automatically taken to the label printing screen when it is done.
```

Comment:
Some odd word wrapping on these two lines, but this text is great!

Reply:
Some auto formatting thing :(